lib/index.js

```diff
@@ -19,6 +19,19 @@ $(document).ready(() => {
   })
 })
 
+const filterFoods = () => {
+  var filter = $('#food-filter-input').val().toUpperCase()
+  var foodNameNodes = $('.food-item-name')
+  for(var i = 0; i < foodNameNodes.length; i++) {
```

Comment:
since foodNameNodes is a collection, we have the array prototype .forEach available.

I find that the forEach is more readable than the `for` syntax.

`
foodNameNodes.forEach( food => {
 if (food.text().toUpperCase().indexOf(filter) !== -1) {
   $(name.parent()[0]).css('display', '')
   } else {
   $(name.parent()[0]).css('display', 'none')
   }
})
`

Reply:
I tried to take that approach of iterating through each food item/match and building on top of that comparison against the filter and the .forEach and .each functions weren't playing nicely with the collection of foodNameNodes that we collect from the window.

I agree that there may be other better ways to do this, but it's functional for now and I think if we would like to revisit this we can do so on a refactoring branch.

Reply:
Thanks for the update. That works for me!